### DISCUSSION TITLE
Still haven’t

### DISCUSSION THREAD

**Discussion by Mrconkilla** [community/community#36284] [Other Feature Feedback, Questions, & Ideas]
Can someone tell me what should I do when my bitcoin is not showing up in my wallet I noticed a few times that the server always disconnects on it own, then it  connects  back, but for some reason does anybody have a solution for me

**Comment by ossanna16** [comment]
Hi @Mrconkilla,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Bitcoin. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
